pkg/configtx/capabilities.go

```diff
@@ -57,8 +51,10 @@ func (c *ConfigTx) ApplicationCapabilities() ([]string, error) {
 	return capabilities, nil
 }
 
-// AddChannelCapability adds capability to the provided channel config.
-func (c *ConfigTx) AddChannelCapability(capability string) error {
+// SetChannelCapability adds capability to the provided channel config.
+// If the provided capability already exist in current configuration, this action
+// will be a no-op.
+func (c *ConfigTx) SetChannelCapability(capability string) error {
```

Comment:
Similar comment to the one for `AddAnchorPeer` ... we are not really setting a single value here but rather adding this